index.rst

```diff
@@ -902,15 +817,28 @@ In the event that a controller in the cabinet needs power cycling remotely, this
 
 AT Dome Communication Loss
 --------------------------
-If during operation the dome controllers lose connection, which is seen either from the software, or the push-buttons fail to work, then this procedure must be followed. The dome has two types of communication failsures
-
-* The two cRIOs lose communication with each other (notably the cRIO in the rotating part of the enclosure loses connection with the bottom box and may be blocking the connection). If the CSC is connected and in disabled or enabled state, then this will be shown in the `scbLink` event (must verify). Also, this can be seen in the Main Box Dome Control LabVIEW Remote on the ATMCS machine as the `TopComms` light in the bottom left corner.
-
-  * Press the reset button on the cRIO inside the electrical cabinet on the rotating part of the dome (near the lower shutter) to resolve this issue
-
-* The Main cRIO (located in the dome electrical cabinet on the first floor) is not correctly releasing the TCP/IP connection. This can be observed by being able to ping the box but not open a telnet connection (port 17310). Also, the HostComms light will be illuminated in the Main Box Dome Control LabVIEW remote.
+If during operation the dome controllers lose connection, which is seen either
+from the software, or the push-buttons fail to work, then this procedure must
+be followed. The dome has two types of communication failsures
+
+* The two cRIOs lose communication with each other (notably the cRIO in the
+rotating part of the enclosure loses connection with the bottom box and may be
+blocking the connection). If the CSC is connected and in disabled or enabled
+state, then this will be shown in the `scbLink` event (must verify). Also, t
+his can be seen in the Main Box Dome Control LabVIEW Remote on the ATMCS
+machine as the `TopComms` light in the bottom left corner.
+
+  * Press the reset button on the cRIO inside the electrical cabinet on the
+    rotating part of the dome (near the lower shutter) to resolve this issue
+
+* The Main cRIO (located in the dome electrical cabinet on the first floor) is
```

Comment:
This has been fixed.

Reply:
has it really? I think we noticed that it has not....